chpl/chpl-service/src/main/java/gov/healthit/chpl/caching/ListingSearchCacheableAspect.java

```diff
@@ -0,0 +1,37 @@
+package gov.healthit.chpl.caching;
+
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+import lombok.extern.log4j.Log4j2;
+import net.sf.ehcache.CacheManager;
+import net.sf.ehcache.Element;
+
+@Component
+@Aspect
+@Log4j2
+public class ListingSearchCacheableAspect {
+    private static final String COLLECTIONS_SEARCH_KEY = "collections search";
+
+    private CacheManager cacheManager;
+
+    @Autowired
+    public ListingSearchCacheableAspect(CacheManager cacheManager) {
+        this.cacheManager = cacheManager;
+    }
+
+    @Around("@annotation(ListingSearchCacheable)")
+    public Object logExecutionTime(ProceedingJoinPoint joinPoint) throws Throwable {
```

Comment:
I'm not sure if I'm missing something or the method name is misleading? I don't see anything being logged, though it's called "logExecutionTime". 
I forgot why we need this annotation and we can't use the regular @ Cacheable annotation on the ListingSearchManager. Can you explain?